lldb/source/Plugins/ExpressionParser/Swift/SwiftExpressionParser.cpp

```diff
@@ -516,6 +516,9 @@ static void AddRequiredAliases(Block *block, lldb::StackFrameSP &stack_frame_sp,
       llvm::cast<TypeSystemSwift>(imported_self_type.GetTypeSystem())
           ->GetReferentType(imported_self_type.GetOpaqueQualType());
 
+  if (!llvm::isa<TypeSystemSwift>(imported_self_type.GetTypeSystem()))
+    return;
+
```

Comment:
I think we should use dyn_cast_or_null instead to guard against a nullptr as well.

Reply:
done